src/deepsparse/benchmark/benchmark_model.py

```diff
@@ -392,13 +400,24 @@ def benchmark_model(
         "benchmark_result": benchmark_result,
     }
 
+    parsed_export_dict = _parse_export_dict_engine_key(export_dict)
+
     # Export results
     if export_path:
         _LOGGER.info("Saving benchmark results to JSON file at {}".format(export_path))
         with open(export_path, "w") as out:
-            json.dump(export_dict, out, indent=2)
+            json.dump(parsed_export_dict, out, indent=2)
 
-    return export_dict
+    return parsed_export_dict
+
+
+def _parse_export_dict_engine_key(payload: Dict) -> Dict:
+    """Extract metadata from str(Model)"""
+
+    formatted_metadata = "engine: " + payload["engine"].replace("\t", "").replace(
+        ":", "", 1
+    )
+    return {**payload, **yaml.load(formatted_metadata, Loader=Loader)}
```

Comment:
could we use dict.update here instead of unpacking? I think it would make the intent more clearer

Reply:
Is dict pass by reference? I did this so i wont mutate if pass by ref